jbmc/src/java_bytecode/bytecode_info.cpp

```diff
@@ -15,86 +15,139 @@ Author: Daniel Kroening, kroening@kroening.com
 // clang-format off
 struct bytecode_infot const bytecode_info[]=
 {
-{ "aaload",         0x32, ' ', 2, 1, ' ' }, // arrayref, index → value; load onto the stack a reference from an array  NOLINT(*)
-{ "aastore",        0x53, ' ', 3, 0, ' ' }, // arrayref, index, value →; store into a reference in an array  NOLINT(*)
+{ "nop",            0x00, ' ', 0, 0, ' ' }, // [No change]; perform no operation  NOLINT(*)
```

Comment:
Given anyone using this table already has a bytecode, the second field is useless now

Reply:
Keeping for audit!